backupctl/backupctl.py

```diff
@@ -301,6 +301,7 @@ def remove(hist, pool, customer, vault=None):
     else:
         fs = os.path.join(pool, customer)
     zfs.remove_filesystem(fs)
+    os.rmdir(fs)
```

Comment:
Wrap in `isdir`:
```python
if os.path.isdir(fs):
    os.rmdir(fs)
```
Otherwise fails for missing directories

Reply:
Good idea, even if zfs mountpoints should _never_ be a file

Reply:
This is not about it being a file, but about it being an *existing* directory. And that's exactly what isdir checks for.